ports/harfbuzz/portfile.cmake

```
@@ -100,6 +100,9 @@ configure_file("${CMAKE_CURRENT_LIST_DIR}/harfbuzzConfig.cmake.in"
 vcpkg_list(SET TOOL_NAMES)
 if("glib" IN_LIST FEATURES)
     vcpkg_list(APPEND TOOL_NAMES hb-subset hb-shape hb-ot-shape-closure hb-info)
+    if("cairo" IN_LIST FEATURES)
+        vcpkg_list(APPEND TOOL_NAMES hb-view)
+    endif()
 endif()
```

```suggestion
if("glib" IN_LIST FEATURES)
    vcpkg_list(APPEND TOOL_NAMES hb-subset hb-shape hb-ot-shape-closure hb-info)
endif()
if("cairo" IN_LIST FEATURES)
    vcpkg_list(APPEND TOOL_NAMES hb-view)
endif()
```
According to below comment:
`'hb-view' will only be built if the optional harfbuzz feature 'cairo' is available.`

So, you cannot add your changes under the condition where `if("glib" IN_LIST FEATURES)` is set to true.

---

It's strange. After applying my suggestions locally, I encountered an error when installing `harfbuzz[cairo]:x64-windows`:
```
CMake Error at scripts/cmake/vcpkg_copy_tools.cmake:36 (message):
  Couldn't find tool "hb-view"
```
So I'd like to clarify with you: is `hb-view` built only when the `cairo` feature is enabled?"

---

Sorry for being inaccurate in my comment, you'll need both 'glib' and 'cairo' for building 'hb-view', hence my suggested change still stands.
See
https://github.com/harfbuzz/harfbuzz/blob/fde1c3692a4a1e45ee48954fb486ae6c036a5687/util/meson.build#L23

---

I have updated this specific comment in
https://github.com/microsoft/vcpkg/issues/34605
and added 'glib' to it

---

Thanks for your reply!